services/ftp/ftp_service.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import tempfile
```

Comment:
import tempfile should go after import os and probably after import shutil according to Nadav's coding conventions #3.d (https://cymmetria.atlassian.net/wiki/spaces/MAZ/pages/316440604/Coding+conventions+-+Python)

Reply:
done